### PR TITLE
Expand c2s suite a little bit

### PR DESCRIPTION
**Pull request by ppikula**

### CODE REVIEW

**Comment by pzel** [comment]
Looks good, though maybe we could delete the unused test cases?

File: apps/ejabberd/test/ejabberd_c2s_SUITE.erl [hunk 1]
```diff
@@ -23,12 +31,56 @@ c2s_start_stop_test(_) ->
     %% then
     ?eq(false, erlang:is_process_alive(C2SPid)).
 
+legacy_auth_stream_feature(_) -> ok.
```
Comment (pzel) [inline apps/ejabberd/test/ejabberd_c2s_SUITE.erl]: unused

File: apps/ejabberd/test/ejabberd_c2s_SUITE.erl [hunk 2]
```diff
@@ -23,12 +31,56 @@ c2s_start_stop_test(_) ->
...
+
+sasl_auth_stream_feature(_) -> ok.
```
Comment (pzel) [inline apps/ejabberd/test/ejabberd_c2s_SUITE.erl]: unused

**Comment by michalwski** [comment]
Or implement them :)

**Comment by ppikula** [comment]
It is Simon's homework ;)

**Comment by michalwski** [comment]
We'll be merged by #637

**Pull request closed without merging**